Doc/library/io.rst

```diff
@@ -55,7 +55,7 @@ the backing store is natively made of bytes (such as in the case of a file),
 encoding and decoding of data is made transparently as well as optional
 translation of platform-specific newline characters.
 
-The easiest way to create a text stream is with :meth:`open()`, optionally
+The easiest way to create a text stream is with :meth:`~io.open`, optionally
```

Comment:
I am not sure this one is a problem; built-in open is io.open after all!  We certainly don’t recommend that people must import io and use io.open.

Reply:
```suggestion
The easiest way to create a text stream is with :meth:`open()`, optionally
```